### PR TITLE
tried but failed this one

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/queueWithStacks/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# stacks and queues
+
+<!-- Short summary or background information -->
+
+## Challenge
+
+create a pseudoQueue using stack
+ not sure what this means
```
Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/README.md:8]: Hint: You want to use stacks to implement a queue

Somewhere you can use to study:
https://www.youtube.com/watch?v=AN0axYeLue0
